### PR TITLE
[BEAM-8684] Google proto and google-services library upgrades

### PR DESCRIPTION
**Pull request by suztomo**
https://issues.apache.org/jira/browse/BEAM-8684

Google proto and google-services library upgrades.

# Linkage Check Result

The following modules use the affected libraries ([command gist](https://gist.github.com/suztomo/80ec74f68c82ec233d1b207274899814))

## beam-sdks-java-extensions-google-cloud-platform-core
- before: https://gist.github.com/suztomo/cd6669b8f6c0e35e1053a13f4cf46f97
- after: https://gist.github.com/suztomo/8734f6fd5d1eddf851d59d6f2a4aa93d (No change)

## beam-sdks-java-io-google-cloud-platform
- https://gist.github.com/suztomo/ee554f3187470d93c1cc265e1540496e
- after: https://gist.github.com/suztomo/c4e7353366068ac140c1af0d85fca401 (No change)

## beam-runners-google-cloud-dataflow-java
- before: https://gist.github.com/suztomo/9829d734c11d311165487e4d77088a1a
- after: https://gist.github.com/suztomo/1645bd8e5fef51536288d96751322b4e (No change)

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [x] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_SparkStructuredStreaming/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python2/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python2_PVR_Flink_Cron/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)
XLang | --- | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/) | --- | --- | ---

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by lukecwik** [comment]
What about the before linkage checker results?

**Comment by suztomo** [comment]
I just added them. Now running diffs.

File: buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy [hunk 1]
```diff
@@ -365,11 +365,10 @@ class BeamModulePlugin implements Plugin<Project> {
     def cassandra_driver_version = "3.8.0"
     def classgraph_version = "4.8.56"
     def gax_version = "1.52.0"
-    def generated_grpc_beta_version = "0.44.0"
     def generated_grpc_ga_version = "1.83.0"
-    def generated_grpc_dc_beta_version = "0.27.0-alpha"
+    def generated_grpc_dc_beta_version = "0.29.0-alpha"
```
Comment (suztomo) [inline buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy]: This is not the latest but 0.30.0-alpha requires code change. Stopping at 0.29.0-alpha for simplicity.

**Comment by suztomo** [comment]
R: @lukecwik 

Ready for review. There's no difference in linkage checks.

**Comment by suztomo** [comment]
Java Portability API precommit failed:
```
17:51:37 Execution failed for task ':runners:google-cloud-dataflow-java:buildAndPushDockerContainer'.
17:51:37 > Process 'command 'docker'' finished with non-zero exit value 1
```

This should be a transient error.

**Comment by aaltay** [comment]
Run JavaPortabilityApi PreCommit

**Comment by suztomo** [comment]
@aaltay Your email client might hide the following 6 commands in my email (my Gmail did). Would you run them?

Run Java PostCommit
Run Java HadoopFormatIO Performance Test
Run BigQueryIO Streaming Performance Test Java
Run Dataflow ValidatesRunner
Run Spark ValidatesRunner
Run SQL Postcommit

**Comment by aaltay** [comment]
Run Java PostCommit

**Comment by aaltay** [comment]
Run Java HadoopFormatIO Performance Test

**Comment by aaltay** [comment]
Run BigQueryIO Streaming Performance Test Java

**Comment by aaltay** [comment]
Run Dataflow ValidatesRunner

**Comment by aaltay** [comment]
Run Spark ValidatesRunner

**Comment by aaltay** [comment]
Run SQL Postcommit

**Comment by suztomo** [comment]
Java HadoopFormatIO Performance Test failed. This is critical. I missed to check linkage errors of sdks:java:io:hadoop-format:

```
19:58:37 > Task :sdks:java:io:hadoop-format:integrationTest FAILED
19:58:37 
19:58:37 org.apache.beam.sdk.io.hadoop.format.HadoopFormatIOIT > writeAndReadUsingHadoopFormat FAILED
19:58:37     java.lang.NoSuchMethodError: com.google.cloud.ServiceOptions.getQuotaProjectId()Ljava/lang/String;
```

I will investigate further next week.


——
Run JavaPortabilityApi PreCommit　failed:

```
19:45:37 > Task :runners:google-cloud-dataflow-java:buildAndPushDockerContainer FAILED
19:45:37 Error response from daemon: No such image: apachebeam/java_sdk:2.20.0-SNAPSHOT
19:45:37 
19:45:37 > Task :runners:google-cloud-dataflow-java:cleanUpDockerImages FAILED
19:45:37 Error: No such image: us.gcr.io/apache-beam-testing/java-postcommit-it/java:20200118004259
```

**Comment by suztomo** [comment]
Will investigate the NoSuchMethodError.

**Comment by suztomo** [comment]
com.google.cloud:google-cloud-bigquery:1.102.0 needs google-cloud-core 1.91.0

**Pull request closed without merging**